roxie/ccd/ccdmain.cpp

```diff
@@ -446,6 +446,9 @@ class CHpccProtocolPluginCtx : implements IHpccProtocolPluginContext, public CIn
 static std::vector<RoxieEndpointInfo> myRoles;
 static std::vector<std::pair<unsigned, unsigned>> agentChannels;
 
+extern void *leakChecker;
```

Comment:
not sure this line is needed.

Reply:
Intention was to head off any (future) compiler deciding that the assignment was never used anywhere...